### PR TITLE
Fix assline NPE

### PR DESCRIPTION
**Pull request by Glease**
https://github.com/GTNewHorizons/GT5-Unofficial/commit/a07d8b58642eb9161479dcb3a285ad9979a74d2c#commitcomment-61224376

### CODE REVIEW

**Comment by GlodBlock** [comment]
didn't alk say it will update the old data stick? why it will return null recipe?

**Comment by Glease** [comment]
Presumbly some junk stuffed into assline or data access hatch, or datastick contains something that no longer have an assline recipe.

**Review by GlodBlock — APPROVED** [review]

File: src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_AssemblyLine.java [hunk 1]
```diff
@@ -175,7 +175,10 @@ public boolean checkRecipe(ItemStack aStack) {
         
         nextDataStick:
         for (ItemStack tDataStick : tDataStickList) {
-        	GT_Recipe_AssemblyLine aFoundRecipe = GT_AssemblyLineUtils.findAssemblyLineRecipeFromDataStick(tDataStick);        	
+        	GT_Recipe_AssemblyLine aFoundRecipe = GT_AssemblyLineUtils.findAssemblyLineRecipeFromDataStick(tDataStick);
+
+            if (aFoundRecipe == null)
+                continue;
```
Comment (bombcar) [inline src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_AssemblyLine.java:181]: can we wrap this in { }

Comment (Glease) [reply src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_AssemblyLine.java:181]: Let's not get into this endless nitpicking

**Pull request merged**